bot/commands.py

```diff
@@ -51,6 +51,9 @@ def check(msg):
             if len(inputs) == 2:
                 uid, emoji_id = inputs
 
+                # Strip the <@> characters from the uid
+                uid = uid.strip('<@!>')
```

Comment:
You want to scrip "!" too?

Reply:
> You want to scrip "!" too?

Discord may include ! if the user has a nickname to my understanding so I wanted to strip ! if it is included as well.